fs/diff_linux.go

```diff
@@ -0,0 +1,92 @@
+package fs
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/pkg/errors"
+	"github.com/stevvooe/continuity/sysx"
+)
+
```

Comment:
Will link to the OCI layer format in the archive, this will be specifically for calculating diffs on AUFS filesystems